scripts/auth.ps1

```diff
@@ -0,0 +1,46 @@
+param (
+    [string]$server = "https://localhost:5001"
+)
+
+# Ignoring self-signed certificates
+add-type @"

```

Comment:
Hah, very clever solution, however I am strongly against with using it, especially in our prod environment (AWS).
First of all, it can lead some security risks which is very dangerous and should be strongly forbidden.
Secondly, as we can see, you use such trick in other scripts as well. I can assume that default settings will still remain.
I admit that did not have much time to help you with SSL due to bug fixing client side, my work and other projects, however first thing I will do is looking for the right solution to unblocking you. Cheers!

Reply:
I can remove this block of code, but then this scripts will become useless until you get a trusted certificate from CA. My suggestion is remove this block only after you get a certificate.

Reply:
Agree, lets do this way.